### PR TITLE
Add seminar 12 nn part-1

### PR DESCRIPTION
**Pull request by Radi4**

### CODE REVIEW

**Comment by shestakoff** [comment]
1) Надо бы сделать не keras а tensorflow.keras
2) Мне не нравится, что нет нормализации данных в примере с keras
3) Можно ли сделать пример не с sequential сеткой а c functional api?
4) dropout?
